src/app/public/modules/tabs/tab-button.component.html

```diff
@@ -10,9 +10,10 @@
     'sky-btn-tab-wizard': tabStyle === 'wizard',
     'sky-btn-tab-selected': active,
     'sky-btn-tab-disabled': disabled,
-    'sky-tab-btn-closeable': closeable,
-    'sky-margin-inline-sm':
-      (themeSvc?.settingsChange | async)?.currentSettings?.theme?.name === 'modern'
```

Comment:
You can remove `@Optional() public themeSvc?: SkyThemeService` from the constructor in the `ts` file.